manifests/install.pp

```diff
@@ -126,12 +126,12 @@
   if $manage_package {
     $_install_status = $::facts['virtualbox_version'] ? {
       Pattern["^${version}"] => 'already-installed',
-      Pattern['^5']          => 'upgrade',
+      Pattern['^6']          => 'upgrade',
```

Comment:
If it starts with `6` but doesn't match the complete version it's an upgrade??  What happens to users who currently have 5 installed?  Do they get upgraded to 6?  What if they don't want to?

Reply:
updated